specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```diff
@@ -0,0 +1,9155 @@
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Deletes an integrationRuntime",
+        "operationId": "IntegrationRuntimes_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IActionResult"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_Delete": {
+            "$ref": "./examples/IntegrationRuntimes_Delete.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Creates or Updates an instance of a integrationRuntime",
+        "operationId": "IntegrationRuntimes_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntime"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntime"
+            }
+          },
+          "201": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntime"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_CreateOrUpdate": {
+            "$ref": "./examples/IntegrationRuntimes_CreateOrUpdate.json"
+          }
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}/status": {
+      "get": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Gets an integrationRuntimeStatus",
```

Comment:
expect a better description.

e.g. I've no idea why status is that important to worth a dedicated endpoint (instead of included in "/integrationruntimes/{integrationRuntimeName}" resource).

Maybe this is already talked in meeting, but to a user new to your REST API, the description is what they got.

Reply:
Agree that this seems unnecessary. Why not just include status in the a get on the resource?

Reply:
From what I saw in Scanning and DataFactory, `IntegrationRuntime` object is vastly different from `IntegrationRuntimeStatus`. Status is periodically polled by the browser where as IntegrationRuntime_Get is not. This might be why DataFactory had same pattern and a separate endpoint. UI was also calling this the same way and I didn't want to introduce a new change when they switched over from calling DataFactory to calling Scan. 

Let me know if that answers the question.